app/models/user.rb

```diff
@@ -1,7 +1,6 @@
 # frozen_string_literal: true
 class User < ActiveRecord::Base
   has_many :versions, inverse_of: :user
```

Comment:
I'm confused by the model relationship between User and Version - why does a version belong to an individual user? Is it just to track who happened to create that version, or is there some limited ownership/visibility of specific versions?